release-notes/swan-lake-release-notes/swan-lake-2202.2.0-release-note.md

```diff
@@ -115,6 +115,46 @@ To view bug fixes, see the [GitHub milestone for 2201.2.0 (Swan Lake)](https://g
 
 #### New features
 
+##### SemVer validator CLI tool [Experimental] 
+Introduced the `bal semver` CLI command, which attempts to validate <a href="https://semver.org/">SemVer</a> compatibility of the local package changes against any previously published version(s) in Ballerina Central. Currently the tool can be used to 
+- list-down the source code differences(along with compatibility impact) between the local and any published versions in Ballerina central
+- suggest the new package version based on the compatibility impact of source code changes
+
+The below example demonstrates semver validator tool functionalities on the `ballerina/io` package.
+
+i. version suggestions
+```
+$bal semver             
+checking for the latest compatible release version available in central...
+
+current version: 1.2.2-SNAPSHOT
+compatibility impact (compared with the release version '1.2.1'): backward-incompatible changes detected
+suggested version: 2.0.0
+```
+ii. version suggestions with the list of detected source code changes
```

Comment:
```suggestion
ii. version suggestions with the list of detected source code changes
```
Shall we make these sub headings?

Reply:
https://github.com/ballerina-platform/ballerina-release/pull/2070#discussion_r929703854

Reply:
+1 to keep as normal bullet points (not numbered).

Reply:
Added with https://github.com/ballerina-platform/ballerina-release/pull/2070/commits/ca7bd413e820b182031b23a29e2bed6514e19eff